apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.Tests/SpannerDbTypeTests.ValueConversions.cs

```diff
@@ -54,13 +54,14 @@ public enum TestType
             { "BoolField", SpannerDbType.Bool, true },
             { "DateField", SpannerDbType.Date, new DateTime(2017, 1, 31) },
             { "TimestampField", SpannerDbType.Timestamp, new DateTime(2017, 1, 31, 3, 15, 30) },
-            { "NumericField", SpannerDbType.Numeric, SpannerNumeric.MaxValue }
+            { "NumericField", SpannerDbType.Numeric, SpannerNumeric.MaxValue },
+            { "JsonField", SpannerDbType.Json, "some-json-string" }
```

Comment:
Is the value of a JSON field meant to be a valid JSON token? If so, I think we should make the test reflect that. Just putting it in double quotes would make it valid for a JSON string token.

Reply:
I've changed it to an empty JSON string (`{}`)